signer/src/storage/postgres.rs

```diff
@@ -42,6 +42,56 @@ fn contract_transaction_kinds() -> &'static HashMap<&'static str, TransactionTyp
     CONTRACT_FUNCTION_NAME_MAPPING.get_or_init(|| CONTRACT_FUNCTION_NAMES.into_iter().collect())
 }
 
+/// A type used for storing transactions in the stacks_transactions table
+#[derive(Debug, serde::Serialize)]
+struct StacksTx {
```

Comment:
I guess this should belong in the `model.rs` module right? Or is this just postgres specific?

Reply:
It's postgres specific.